File: Easy/Python/2833. Furthest Point From Origin.py
class Solution(object):
    def furthestDistanceFromOrigin(self, moves):
        countL = 0
        countR = 0
        countBlanks = 0

        for i in range(len(moves)):
            if moves[i] == 'L':
                countL += 1
            elif moves[i] == 'R':
                countR += 1
            else:
                countBlanks += 1
        
        return abs(countL - countR) + countBlanks

File: Easy/Java/2833. Furthest Point From Origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int countL = 0;
        int countR = 0;
        int countBlanks = 0;
        
        for(int i = 0; i < moves.length(); i++){
            if(moves.charAt(i) == 'L')
                countL++;
            else if(moves.charAt(i) == 'R')
                countR++;
            else
                countBlanks++;
        }

        return Math.abs(countL - countR) + countBlanks;
    }
}
